.github/workflows/check-links.yml

```diff
@@ -61,6 +62,8 @@ jobs:
           --exclude "^https://www.adobe.com/jp/print/postscript/pdfs/PLRM.pdf"
           --exclude "^https://www.generic-mapping-tools.org/remote-datasets/%s$"
           --exclude "^https://www.pygmt.org/%7B%7Bpath%7D%7D"
+          --exclude "^https://github.com/GenericMappingTools/pygmt/releases/download/%7B%7Bpath%7D%7D/pygmt-docs.pdf"
+          --exclude "^https://github.com/GenericMappingTools/pygmt/releases/download/%7B%7Bpath%7D%7D/pygmt-docs.zip"
```

Comment:
Move to line 58?

Reply:
Makes sense! Done in commit https://github.com/GenericMappingTools/pygmt/pull/3875/commits/f24ac8d1060db1f8c51e55ca98877a0ae9f89e3c.